test/test_translator.py

```diff
@@ -138,8 +209,78 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_null_parameter() :
+
+    test_polyline= None
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    expected_coordinates= None
+    assert  test_coordinates == expected_coordinates
 
-def test_get_road_direction():
+def test_parse_polyline_invalid_data() :
+    test_polyline= 'invalid' 
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    expected_coordinates= []
+    assert  test_coordinates == expected_coordinates
+
+class TestTranslator(unittest.TestCase):
```

Comment:
If we want to use the class, this should be a wrapper for all the methods in this file, not just the individual test.

Reply:
i updated the test function.It should be fixed the problem